python/ray/tests/test_basic_5.py

```diff
@@ -78,5 +78,33 @@ def foo(self):
     assert ray.get(worker_2.foo.remote()) is None
 
 
+def test_internal_kv(ray_start_regular):
+    import ray.experimental.internal_kv as kv
+    assert kv._internal_kv_get("k1") is None
+    assert kv._internal_kv_put("k1", "v1") is False
```

Comment:
It seems unintuitive that _internal_kv_put returns False when inserting a new key, and True otherwise. I wonder if it can be changed.

Reply:
I'd prefer not to change the API semantics in this PR. It needs more investigation.

Reply:
Sounds good.